packages/babel-parser/src/parser/expression.js

```diff
@@ -500,7 +515,15 @@ export default class ExpressionParser extends LValParser {
       this.next();
       return this.parseAwait(startPos, startLoc);
     }
-
+    if (this.isContextual("module")) {
+      const lookahead = this.lookahead();
+      if (
+        lookahead.type === tt.braceL &&
```

Comment:
We can use `this.lookaheadCharCode() === charCodes.rightCurlyBrace` and avoid the expensive `this.lokahead()`.

Reply:
Also we can avoid lookahead via using new `hasFollowingLineBreak`, thank you. 370953a